backend/package.json

```diff
@@ -36,6 +44,7 @@
     "jsonwebtoken": "^8.5.1",
     "jwks-client": "^1.4.0",
     "knex": "^2.0.3",
+    "module-alias": "^2.2.2",
```

Comment:
just wondering what vetting you did for this module, i like the concept, but this is another vector that i'm not 100% sure i want to add.